proposals/draft/Draft - JSONSchema Version Guidance.md

```diff
@@ -0,0 +1,80 @@
+## Status
+
+| header | header |
+| ------ | ------ |
+| State | Draft  |
+| Issue Link | https://github.com/MeltanoLabs/Singer-Working-Group/issues/17 |
+| Discussion Thread(s) | (optional link) |
+| Created | 2021-10-25 |
+
+-----------------------
+
+## Proposal
+
+### TL;DR Overview
+
+We should provide guidance for recommended and supported JSON Schema versions to tap and target developers which 
+
+### What specific change do you propose to make?
+
+We should document in the Singer Spec that Singer taps and targets expect **Draft 04** version of the JSON Schema spec.
```

Comment:
To expand a bit on this, we could even go as far as saying something like

Targets default to expecting Draft04 to be sent. To use a different json schema version use the $schema keyword.  (To date nothing uses anything past Draft04 to my knowledge but I'm probably wrong based on how many taps/targets there are) 

Taps should default to sending Draft04 unless they have a reason to use a later version of Json schema.

Reply:
Should we also include a why here? ie Draft04 is widely used and works for the vast majority of use cases as of today.